engine/src/lib.rs

```diff
@@ -78,32 +77,6 @@ impl ServalEngine {
         let stdout = WritePipe::new_in_memory();
         let stderr = WritePipe::new_in_memory();
 
-        // Link the experimental HTTP support
```

Comment:
Consider removing the [HTTP-related permissions](https://github.com/serval/serval-mesh/blob/c76a430924c0008a9f30c3ea1ed9de06ff97bd6f/utils/src/structs/mod.rs#L238-L239) and the [http-test sample project](https://github.com/serval/wasm-samples/tree/main/http-test).